ember-slugify/package.json

```diff
@@ -0,0 +1,83 @@
+{
+  "name": "ember-slugify",
+  "description": "Library to slugify your strings within Ember.",
+  "version": "4.0.0",
+  "license": "MIT",
+  "author": {
+    "name": "Dazzling Fugu",
+    "url": "https://github.com/DazzlingFugu"
+  },
+  "homepage": "https://github.com/DazzlingFugu/ember-slugify",
+  "repository": "https://github.com/DazzlingFugu/ember-slugify",
+  "bugs": "https://github.com/DazzlingFugu/ember-slugify/issues",
+  "keywords": [
+    "accent",
+    "accents",
+    "diacritics",
+    "ember",
+    "emberjs",
+    "ember-addon",
+    "simple-pinyin",
+    "slug",
+    "slugify",
+    "string"
+  ],
+  "files": [
+    "addon-main.js",
+    "dist"
+  ],
+  "scripts": {
+    "lint": "npm-run-all --print-name --aggregate-output --continue-on-error --parallel \"lint:!(fix)\"",
+    "lint:fix": "npm-run-all --print-name --aggregate-output --continue-on-error --parallel \"lint:*:fix\"",
+    "lint:hbs": "ember-template-lint .",
+    "lint:hbs:fix": "ember-template-lint . --fix",
+    "lint:js": "eslint . --cache",
+    "lint:js:fix": "eslint . --fix",
+    "build": "rollup --config",
+    "prepublishOnly": "rollup --config",
+    "start": "rollup --config --watch"
+  },
+  "dependencies": {
+    "@embroider/addon-shim": "^1.8.4",
+    "emoji.json": "^13.1.0",
+    "simple-pinyin": "^4.0.0",
+    "slugify": "^1.6.5"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.21.0",
+    "@babel/plugin-proposal-class-properties": "^7.18.6",
+    "@babel/plugin-proposal-decorators": "^7.21.0",
+    "@embroider/addon-dev": "^3.0.0",
+    "@rollup/plugin-babel": "^6.0.3",
+    "eslint": "^8.16.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-plugin-ember": "^11.1.0",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-prettier": "^4.2.1",
+    "eslint-plugin-qunit": "^7.3.1",
+    "npm-run-all": "^4.1.5",
+    "prettier": "^2.7.1",
+    "rollup": "^3.17.3"
+  },
+  "engines": {
+    "node": "14.* || 16.* || >= 18"
+  },
+  "ember": {
+    "edition": "octane"
+  },
+  "ember-addon": {
+    "main": "addon-main.js",
+    "type": "addon",
+    "version": 2,
+    "app-js": {
+      "./helpers/remove-diacritics.js": "./dist/_app_/helpers/remove-diacritics.js",
+      "./helpers/slugify.js": "./dist/_app_/helpers/slugify.js"
```

Comment:
For personal knowledge, when migrating to V2 addon do we have to list all files exported by the addon (including js/hbs ones?)